src/super_gradients/common/object_names.py

```diff
@@ -35,6 +35,16 @@ class Metrics:
     PIXEL_ACCURACY = "PixelAccuracy"
     POSE_ESTIMATION_METRICS = "PoseEstimationMetrics"
     DETECTION_METRICS_DISTANCE_BASED = "DetectionMetricsDistanceBased"
+    DELTAMETRIC = "DeltaMetric"
```

Comment:
Should we add `Depth` prefix to these metrics? MSE is quite common name and maybe it may cause a confusion if some day we decide to use MSE for image reconstruction